src/LanguageServer/Protocol/Handler/Diagnostics/DiagnosticSources/AbstractWorkspaceDocumentDiagnosticSource.cs

```diff
@@ -79,9 +80,10 @@ AsyncLazy<ILookup<DocumentId, DiagnosticData>> GetLazyDiagnostics()
                                 diagnosticIds: null, shouldIncludeAnalyzer,
                                 // Ensure we compute and return diagnostics for both the normal docs and the additional docs in this project.
                                 static (project, _) => [.. project.DocumentIds.Concat(project.AdditionalDocumentIds)],
-                                includeSuppressedDiagnostics: false,
                                 includeLocalDocumentDiagnostics: true, includeNonLocalDocumentDiagnostics: true, cancellationToken).ConfigureAwait(false);
-                            return allDiagnostics.Where(d => d.DocumentId != null).ToLookup(d => d.DocumentId!);
+
+                            // TODO(cyrusn): Should we be filtering out suppressed diagnostics here?
```

Comment:
>TODO [](http://example.com/codeflow?start=31&length=4)

Just wanted to clarify whether this is an active question or a remnant added while doing this investigate?

Reply:
this is an active question.  will clarify.